cirq/study/sweepable_test.py

```diff
@@ -83,6 +83,18 @@ def test_to_sweeps_iterable_sweeps():
     assert cirq.study.to_sweeps(sweeps) == sweeps
 
 
+def test_to_sweeps_dictionary_of_list():
+    assert cirq.study.to_sweeps({'t': [0, 2, 3]}) == \
+        cirq.study.to_sweeps([{'t': 0}, {'t': 2}, {'t': 3}])
+    assert cirq.study.to_sweeps({'t': [0, 1], 's': [2, 3], 'r': 4}) == \
```

Comment:
I think this case is quite ambiguous.  You are assuming that this is a product, but it could be a zip, i.e.

```
cirq.study.to_sweeps([
            {'t': 0, 's': 2, 'r': 4},
            {'t': 1, 's': 3, 'r': 4},
        ])
```

At the very least, we need to document that multiple lists will be interpreted as a Product and not a Zip.